segmentstore/server/src/test/java/io/pravega/segmentstore/server/TestCacheManager.java

```diff
@@ -9,14 +9,15 @@
  */
 package io.pravega.segmentstore.server;
 
+import io.pravega.segmentstore.storage.cache.CacheStorage;
 import java.util.concurrent.ScheduledExecutorService;
 
 /**
  * Exposes the applyCachePolicy method in the CacheManager.
  */
 public class TestCacheManager extends CacheManager {
```

Comment:
`CacheManagerTests` is unit tests for `CacheManager`
`TestCacheManager` is an extension for `CacheManager` that exposes the `applyCachePolicy` method, so that other unit tests (for other components) that want to verify their cache eviction code can do so in a controlled manner (without having to rely on a timer to do so).